lib/board.rb

```diff
@@ -9,5 +9,11 @@ def initialize
   end
 
   def generate_cells
```

Comment:
This looks great! Method populates hash with coordinate strings as keys, creates 16 pairs, and uses Cell instance as value.